hail/python/hailtop/hailctl/dev/benchmark/compare/compare.py

```diff
@@ -18,15 +18,12 @@ def load_file(path):
     return {x['name']: x for x in js_data}
 
 
-def magnitude(time1, time2):
-    return max(time1 / time2, time2 / time1)
-
-
 def fmt_diff(ratio):
     if ratio < 1:
-        return f'+{1 / ratio:.3f}'
+        return f'+{(1 / ratio - 1):.3f}'
     else:
-        return f'-{ratio:.3f}'
+        return f'-{(ratio - 1):.3f}'
```

Comment:
Personally, I find this confusing, and I have to keep running through in my head what positive vs. negative numbers mean. I think just reporting `ratio` (maybe as a percentage) is perfectly clear. But if you like this better I'm okay with it.

Reply:
yeah, I think you're right.